sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/ServiceBusTriggerAttribute.cs

```diff
@@ -106,12 +107,34 @@ public bool AutoCompleteMessages
             }
         }
 
+        /// <summary>
+        /// Gets or sets the maximum number of messages that will be passed to each function call. This only applies for functions that receive
+        /// a batch of messages.
+        /// </summary>
+        public int MaxMessageBatchSize
+        {
+            get
+            {
+                return _maxMessageBatchSize.HasValue ? _maxMessageBatchSize.Value : 0;
```

Comment:
Should this use the same default that we define in the host.json setting? https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusOptions.cs#L178